R/location_methods.R

```diff
@@ -70,26 +71,22 @@ add_summary_location_row <- function(loc,
       unlist() %>%
       unique()
 
-    col_idx <-
-      resolve_data_vals_idx(
-        var_expr = !!loc$columns,
-        data_tbl = NULL,
-        vals = dt_boxhead_get_vars_default(data = data)
+    columns <-
+      resolve_cols_c(
+        expr = !!loc$columns,
+        data = data
```

Comment:
Let's make sure we have a test case for adding a summary on a column that's hidden: both whether that produces an error or not, and if not, whether unhiding the column will result in the summary being shown.

Reply:
Now done. Left a TODO in the testing code to circle back and test the rendered table with `cols_unhide()` after the PR with that new function is merged.